docker/environment-python3-idr.yml

```diff
@@ -15,9 +12,9 @@ dependencies:
 - python-graphviz=0.13=py_0
 - python-igraph=0.7.1.post6=py36_4
 - scikit-learn=0.21.*
-- seaborn=0.9.0=py36_0
 - tqdm=4.36.1=py_0
+- pip
 # defaults
 - pip:
   - py2cytoscape==0.7.1
```

Comment:
Looks like this is now on conda-forge: https://anaconda.org/conda-forge/py2cytoscape

Reply:
thanks